src/neofs_testlib/env/env.py

```diff
@@ -199,23 +202,69 @@ def persist(self) -> str:
         logger.info(f"Persist env at: {persisted_path}")
         return persisted_path
 
+    @allure.step("Download binaries")
+    def download_binaries(self):
+        logger.info(f"Going to download missing binaries, if any")
+        deploy_threads = []
+        
+        binaries = [
+            (self.neofs_adm_path, "neofs_adm"),
+            (self.neofs_cli_path, "neofs_cli"),
+            (self.neo_go_path, "neo_go"),
+            (self.neofs_ir_path, "neofs_ir"),
+            (self.neofs_node_path, "neofs_node"),
+            (self.neofs_s3_authmate_path, "neofs_s3_authmate"),
+            (self.neofs_s3_gw_path, "neofs_s3_gw"),
+            (self.neofs_rest_gw_path, "neofs_rest_gw"),
+            (self.neofs_http_gw_path, "neofs_http_gw"),
```

Comment:
Why underscores in binary names? They're using dashes in fact.

Reply:
these are not binary names, these are keys from config file that leads to actual data